workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb

```diff
@@ -71,7 +71,7 @@
     "from tfx.components import BigQueryExampleGen\n",
```

Comment:
<ul><li>Move this licensing stuff at the very end of the notebook</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I will make this formatting change in a subsequent PR for the exercises notebook.

Reply:
<p>okay. Make sure that the formatting changes are the same in the exercises and the notebook.</p>